### ISSUE TITLE
Testing accuracy

### ISSUE THREAD

**Issue by ronitkathuria15** [ronitkathuria15/traffic-sign-recognition#2]
To properly measure how the model generalizes to data it hasn’t seen, we must carry out an evaluation on test data that wasn't used in training. The BelgiumTS dataset makes this easy by providing two separate sets, one for training and one for testing. Therefore, we must load the test set, resize the images to 32x32, and then calculate the accuracy.

**Comment by ronitkathuria15** [comment]
This is the relevant part of the code for calculating the inaccuracy:


# Load the test dataset.
test_images, test_labels = load_data(test_data_dir)

# Transform the images, just like we did with the training set.
test_images32 = [skimage.transform.resize(image, (32, 32), mode='constant')
                 for image in test_images]
display_images_and_labels(test_images32, test_labels)

# Run predictions against the full test set.
predicted = session.run([predicted_labels], 
                        feed_dict={images_ph: test_images32})[0]
# Calculate how many matches we got.
match_count = sum([int(y == y_) for y, y_ in zip(test_labels, predicted)])
accuracy = match_count / len(test_labels)
print("Accuracy: {:.3f}".format(accuracy))
